core/src/test/scala/quasar/plugin/postgres/PostgresDestinationSpec.scala

```diff
@@ -628,11 +622,12 @@ object PostgresDestinationSpec extends EffectfulQSpec[IO] with CsvSupport with P
       ("zero" ->> DateTimeInterval.zero) ::
       ("pos" ->> DateTimeInterval.make(3, 1, 14, 600, 342000)) ::
       HNil)
+
   }
 
   val DM = PostgresDestinationModule
 
-  val TestConnectionUrl: String = "postgresql://localhost:54322/postgres?user=postgres&password=postgres"
+  val TestConnectionUrl: String = "postgresql://localhost:5433/postgres?user=postgres&password=secret"
```

Comment:
This needs to be reverted in order for ci to work, not sure why it hasn't been running though.